src/main/java/com/toamistaketracker/ToaMistake.java

```diff
@@ -6,71 +6,87 @@
 
 import java.awt.image.BufferedImage;
 import java.util.function.Function;
+import java.util.Arrays;
 import java.util.EnumSet;
 import java.util.Set;
 
 public enum ToaMistake {
     // Deaths
-    DEATH("Death", (config) -> "", "death.png"), // Chat message handled in deaths below
-    DEATH_HET("Path of Het Death", (config) -> config.deathMessage(), "death-akkha.png"),
-    DEATH_CRONDIS("Path of Crondis Death", (config) -> config.deathMessage(), "death-zebak.png"),
-    DEATH_SCABARAS("Path of Scabaras Death", (config) -> config.deathMessage(), "death-kephri.png"),
-    DEATH_APMEKEN("Path of Apmeken Death", (config) -> config.deathMessage(), "death-baba.png"),
-    DEATH_WARDENS("Wardens Death", (config) -> config.deathMessage(), "death-wardens.png"),
+    DEATH("Death", (config) -> "", "I'm planking!", "death.png"), // Chat message handled in deaths below
```

Comment:
Let's keep the default here as empty string. I misremembered that if we actually have this spit out a non-empty message (which it could if we have to fallback to the default), then it will display both messages when you die (the top-level one and the room-level one below).

Probably shouldn't work that way in the first place, but it's not worth changing now IMO

To get around the duplication issue, and the fact that you need to choose one for the config, let's create a static constant at the top `DEFAULT_DEATH_MESSAGE` and use for all the room deaths (and the death config). If we ever want to have different death messages per room (not really planned as of now) then we can worry about changing that then.